pkg/scheduler/factory_test.go

```diff
@@ -273,6 +273,26 @@ func TestCreateFromConfigWithUnspecifiedPredicatesOrPriorities(t *testing.T) {
 	}
 }
 
+func newIndexer(get func(namespace, name string) (interface{}, error)) cache.Indexer {
+	return &fakeIndexer{get: get}
```

Comment:
is this worth a function?

Reply:
I forgot to do the UT polish which I promised earlier https://github.com/kubernetes/kubernetes/pull/90660/#issuecomment-624984944. Updated.